satellite/satellitedb/overlaycache.go

```diff
@@ -1338,3 +1338,130 @@ func populateUpdateFields(dbNode *dbx.Node, updateReq *overlay.UpdateRequest) db
 
 	return updateFields
 }
+
+// UpdateCheckIn updates a single storagenode with info from when the the node last checked in.
+func (cache *overlaycache) UpdateCheckIn(ctx context.Context, node overlay.NodeCheckInInfo, config overlay.NodeSelectionConfig) (err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	if node.Address.GetAddress() == "" {
+		return Error.New("error UpdateCheckIn: missing the storage node address")
+	}
+
+	switch t := cache.db.Driver().(type) {
+	case *sqlite3.SQLiteDriver:
+		value := pb.Node{
+			Id: node.NodeID,
+			Address: &pb.NodeAddress{
```

Comment:
nit - you can change this to `Address: node.Address` now

Reply:
good catch.